### ISSUE TITLE
Name operators with operator type by default

### ISSUE THREAD

**Issue by todderasesareddot** [fabric-engine/Kraken#468]
When building operators, would be very helpful to name the canvas node based on the type of KLSolver or Preset instead of just the generic "klOp" or "canvasOp".

I have code that appends the solver type to the name upon init.  This seems like the easiest way and the feature can be turned off via argument if you want more control over naming, but this seems like a good default.

I can submit a pull request.  Eric, what do you think?

**Comment by EricTRocks** [comment]
Yeah I can see the usefulness for sure. I think we can try to extend the naming template for the operators to take into consideration the preset or solver type which would allow users to just use the normal naming template.

We just need to add a new token in the `Operator()` class' `getBuildName()` method and be able to query the preset / kl solver name from there. Maybe the token would be `[solverName]`?

**Comment by bhaux** [comment]
we can close this one...
